platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -729,6 +729,14 @@ The `model` and `component` tasks report on the structure of legacy software mod
 Previously, these tasks were automatically added to the project for every build.
 These tasks are now only added to a project when a rule-based plugin is applied (such as those provided by Gradle's support for building native software).
 
+=== Potential breaking changes
+
+==== Upgrade to Eclipse JGit 7.2.1
+
+Eclipse JGit has been updated from 5.13.3 to https://projects.eclipse.org/projects/technology.jgit/releases/7.2.1[7.2.1].
```

Comment:
```suggestion
Eclipse JGit has been updated from 5.13.3 to [7.2.1](https://projects.eclipse.org/projects/technology.jgit/releases/7.2.1).
```

Reply:
I accepted them, but had to revert them because of an error in `checkDeadInternalLinks`:

> ERROR: upgrading_version_8.adoc:736 Markdown-style links are not supported: [7.2.1](https://projects.eclipse.org/projects/technology.jgit/releases/7.2.1